staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -147,3 +149,108 @@ func IsStoredVersion(crd *CustomResourceDefinition, version string) bool {
 	}
 	return false
 }
+
+// GetSchemaForVersion returns the validation schema for given version in given CRD.
```

Comment:
if we're exposing this, doc the possibility of returning `nil, nil` and when an error is returned (applies to all)

Reply:
let me clean up the docs and also func names a bit, while we are at it.

Reply:
done